src/pages/on-chain-governance/_components/ProgressBar.tsx

```diff
@@ -0,0 +1,74 @@
+import BigNumber from "bignumber.js";
+import classNames from "classnames";
+
+interface Progress {
+  totalTime: BigNumber;
+  timeLeft: BigNumber;
+}
+
+export function ProgressBar({
+  votingProgress,
+  submissionProgress,
+  segment = 1,
+}: {
+  votingProgress: Progress;
+  submissionProgress?: Progress;
+  segment?: 1 | 2;
```

Comment:
should progress bar have 3 segment instead of 2

Reply:
will adjust accordingly when requirement is finalized